plugins/jfrog-artifactory/content.yaml

```diff
@@ -0,0 +1,74 @@
+title: JFrog Artifactory
+author: harness
+tags:
+  - publish
+  - artifactory
+logo: artifactory.svg
+repo: https://github.com/harness/drone-artifactory
+image: https://hub.docker.com/r/plugins/artifactory
+license: None
```

Comment:
Can you add the Blue oak license here?

Reply:
https://github.com/drone/drone-plugin-index/pull/389/commits/03902f5c4e3f7966f78063dbfcaeee73db6f665f